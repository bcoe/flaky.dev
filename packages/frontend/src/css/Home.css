.Home {
  background-color: blue;
  padding: 30px;
}