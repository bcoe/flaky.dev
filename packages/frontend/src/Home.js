import React from 'react';
import './css/Home.css';

function Home() {
  return (
    <div className="Home">
      Home Screen
    </div>
  );
}

export default Home;
