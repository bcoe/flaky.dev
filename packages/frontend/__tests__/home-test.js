/* global test, expect */

import React from 'react';
import Home from '../src/home';
import renderer from 'react-test-renderer';

test('renders home screen with login button', () => {
	const component = renderer.create(
		<Home/>
	);
	// See: https://jestjs.io/docs/en/tutorial-react
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
