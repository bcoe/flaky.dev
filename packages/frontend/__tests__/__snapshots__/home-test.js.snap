// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home screen with login button 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <div
        className="fade alert alert-success show"
        role="alert"
      >
        <div
          className="alert-heading h4"
        >
          Welcome to flaky.dev
        </div>
        <p>
          Flaky.dev accepts output from your test runs on GitHub, performs statistical analysis, and opens and closes issues for flaky tests.
        </p>
        <hr />
        <p
          className="mb-0"
        >
          Login with GitHub below to get started.
        </p>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    />
    <div
      className="col"
    >
      <button
        className="btn btn-primary btn-block"
        disabled={true}
        type="button"
      >
        Loadingâ€¦
      </button>
    </div>
    <div
      className="col"
    />
  </div>
</div>
`;
